CONTRIBUTING.md

```diff
@@ -76,7 +76,7 @@ Please note if someone is already assigned or working on the issue, do not try t
 - `no activity` labeled PR are meant to have no activity in the PR from since a while.
 - `blocked` labeled PR are meant to not to go ahead for review.
 - `do not merge` labeled PR are meant to not to merge the PR right now(may be later).
-- [awaiting-approval](https://github.com/CircuitVerse/CircuitVerse/labels/awaiting-approval) labeled PR are meant to be waiting for other core-team members.
+- [awaiting-approval](https://github.com/CircuitVerse/CircuitVerse/labels/awaiting-approval) labeled PR are meant to be waiting for other communtiy members.
```

Comment:
`- [awaiting-approval](https://github.com/CircuitVerse/CircuitVerse/labels/awaiting-approval) labeled PRs are meant to be waiting for other communtiy members.`